docker-compose.yaml

```diff
@@ -453,6 +483,89 @@ services:
       - "18083:8080"
     profiles: [ "itest" ]
 
+  rabbitmq:
+    image: rabbitmq:3.12.2-alpine
+    profiles: [ "opennotificaties" ]
```

Comment:
```suggestion
    healthcheck:
      test: rabbitmq-diagnostics -q ping
      interval: 30s
      timeout: 30s
      retries: 3
    profiles: [ "opennotificaties" ]
```

Reply:
Nice, thanks. Will add. Not sure if it is really needed but looks give it is good to have. I copied this setup from the OpenNotificaties Docker Compose file where they also don't have a healthcheck for RabbitMQ..